graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/NodeIdInputTest.java

```diff
@@ -173,7 +173,8 @@ void typeWithoutTable() {
                 "customer.getPrimaryKey().getFieldsArray())))" +
                         ".from(_a_customer)" +
                         ".where(_iv_nodeIdStrategy.hasId(\"CustomerNode\"",
-                "CustomerNoTable::new))).fetchOne("
+                "CustomerNoTable::new))",
+                ".fetchOne("
```

Comment:
```suggestion
                "queryForQuery_customerNoTable(inRecord, _iv_nodeIdStrategy)).fetchOne("
```
Tror poenget med asserten her var å sjekke at conditionen blir plassert riktig